compiler/pragmas.nim

```diff
@@ -867,7 +871,8 @@ proc singlePragma(c: PContext, sym: PSym, n: PNode, i: var int,
       of wThreadVar:
         noVal(c, it)
         incl(sym.flags, {sfThread, sfGlobal})
-      of wDeadCodeElimUnused: discard  # deprecated, dead code elim always on
+      of wDeadCodeElimUnused: deprecatedPragma(wDeadCodeElimUnused, "dead code elim always on")
+      of wIncompleteStruct: deprecatedPragma(wIncompleteStruct, "see '$1' instead" % wCompleteStruct.canonPragmaSpelling)
```

Comment:
```suggestion
      of wIncompleteStruct: deprecatedPragma(wIncompleteStruct, "every object is now an 'incompleteStruct', unless annotated with '$1'" % wCompleteStruct.canonPragmaSpelling)
```

Reply:
resolved by pointing to manual section (avoids duplication)